dotcom-rendering/src/web/components/Nav/Nav.tsx

```diff
@@ -191,23 +176,12 @@ export const Nav = ({
 				data-component="nav2"
 			>
 				{format.display === ArticleDisplay.Immersive && (
-					<Hide when="above" breakpoint="tablet">
-						<ThemeProvider theme={buttonThemeReaderRevenue}>
-							<PositionButton>
-								<LinkButton
-									priority="primary"
-									size="small"
-									iconSide="right"
-									icon={<SvgArrowRightStraight />}
-									data-link-name="nav2 : support-cta"
-									data-edition={editionId}
-									href={subscribeUrl}
-								>
-									Subscribe
-								</LinkButton>
-							</PositionButton>
-						</ThemeProvider>
-					</Hide>
+					<Island deferUntil="visible" clientOnly={true}>
```

Comment:
And it does seem weird to defer a button until visible when it’s at the top of the page – but not sure I’ve understood that correctly.

Reply:
@undergroundquizscene `dotcom-rendering` renders pages on the server using React, however certain elements on the page will require application state and interactivity in the client. For example this `InteractiveSupportButton` component needs to read a user's cookies to determine the user's supporter status, and whether it should display or not.

"Hydration" is a feature of React, it's the process of using client-side JavaScript to re-render the page in the client and attach state and interactivity to components that require it the client. Partial Hydration is an enhancement where we only re-render those components that need to be hydrated, rather than the entire page.

This page in the docs offers further detail: https://github.com/guardian/dotcom-rendering/blob/c698a74c08222b4c482522be89190daf6604cd58/dotcom-rendering/docs/architecture/027-better-partial-hydration.md

Reply:
> And it does seem weird to defer a button until visible when it’s at the top of the page – but not sure I’ve understood that correctly.

Yeah I'm not sure about this, `deferUntil` is used to defer hydration until necessary, I'm not sure how this component behaves without the `deferUntil="visible"` prop? @LAKSHMIRPILLAI did we try removing this prop?